src/Components/AdvancedSearch/AdvancedSearchModal.stories.local.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import MsalAuthService from '../../Models/Services/MsalAuthService';
+import useAuthParams from '../../../.storybook/useAuthParams';
+import AdvancedSearchModal from './AdvancedSearchModal';
+import { getDefaultStoryDecorator } from '../../Models/Services/StoryUtilities';
+import { IAdvancedSearchProps } from './AdvancedSearch.types';
+import { ComponentStory } from '@storybook/react';
+import { ADTAdapter } from '../../Adapters';
+
+const wrapperStyle = { width: '100%', height: '100vh', padding: 8 };
+
+export default {
+    title: 'Components/AdvancedSearch',
+    component: AdvancedSearchModal,
+    decorators: [getDefaultStoryDecorator<IAdvancedSearchProps>(wrapperStyle)]
+};
+
+type AdvancedSearchModalStory = ComponentStory<typeof AdvancedSearchModal>;
+
+const Template: AdvancedSearchModalStory = (_args) => {
+    const authenticationParameters = useAuthParams();
+
+    return !authenticationParameters ? (
+        <div></div>
+    ) : (
+        <AdvancedSearchModal
+            adapter={
+                new ADTAdapter(
+                    authenticationParameters.adt.hostUrl,
+                    new MsalAuthService(
+                        authenticationParameters.adt.aadParameters
+                    ),
+                    authenticationParameters.storage.blobContainerUrl
+                )
+            }
+            allowedPropertyValueTypes={[
+                'string',
```

Comment:
is this list going to be the only allowed properties for advanced search, can we define this as const in AdvancedSearch.types.ts file? We used to use such const for property types before like `numericPropertyValueTypes`

Reply:
Good suggestion, I'll make this change